src/bots/unvetter.py

```diff
@@ -90,12 +89,14 @@ def execute(self, block: BlockData) -> bool:
 
         return True
 
-    def receive_unvet_messages(self) -> list[UnvetMessage]:
+    def receive_unvet_messages(self) -> Iterable[UnvetMessage]:
```

Comment:
In the return annotation, it is best to describe the type as precisely as possible.
Knowing that there is not just iterable, but list we can, for example, just call len method without any casting.
In the input annotations better to use minimal requirements for values. For example, if we're just iterating over an object, there is no need to accept list, we can accept any iterable object

Reply:
Reverted